### PR TITLE
dotCMS/core#23941 fix clipboard alignment issue

### LINKED ISSUE #23941: Clipboard has UX Issues after Dojo Upgrade
[](https://mrkr.io/s/63d7fde7a189f8eba0938873/0)

### Problem Statement
There are three different fonts and weights of text. (needs re-design)
The text is not aligned between each other or the box below.
Should we add a button with the option of adding an image to the clipboard?

### Steps to Reproduce
1. Edit a content with WYSIWYG field
2. Click on the "Clipboard" button

```[tasklist]
### Acceptance Criteria
- [x] Fix the alignment
- [ ] Use the system font
```

### dotCMS Version
All

**Reported by:** Melissa Rojas Rodríguez (melissa.rojas@dotcms.com)

**Source URL:** [https://demo.dotcms.com/dotAdmin/#/c/c_Rich-Text/b56cfea1-b78a-495f-af35-29de0af8cd0a](https://demo.dotcms.com/dotAdmin/#/c/c_Rich-Text/b56cfea1-b78a-495f-af35-29de0af8cd0a)

**Issue details:** [Open in Marker.io](https://app.marker.io/i/63d7fde7a189f8eba0938876_2d4007537d54bcb7?advanced=1)

<table><tr><td><strong>Device type</strong></td><td>desktop</td></tr><tr><td><strong>Browser</strong></td><td>Chrome 109.0.0.0</td></tr><tr><td><strong>Screen Size</strong></td><td>1352 x 878</td></tr><tr><td><strong>OS</strong></td><td>OS X 13.0.0</td></tr><tr><td><strong>Viewport Size</strong></td><td>1352 x 844</td></tr><tr><td><strong>Zoom Level</strong></td><td>100%</td></tr><tr><td><strong>Pixel Ratio</strong></td><td>@&#8203;2x</td></tr></table>

### PR DESCRIPTION
**Pull request by zulqarnainvd**
### Proposed Changes
* fix clipboard alignment

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **
image|image

### CODE REVIEW

**Comment by fmontes** [comment]
Can you apply the font Roboto so it match the dotCMS font?

**Comment by zulqarnainvd** [comment]
> Can you apply the font Roboto so it match the dotCMS font? you can load it from the dotcms instance.

I already set the `Roboto` font in `skin.min.css` file and it's working fine . do you want to load font using specifically path?

File: dotCMS/src/main/webapp/html/js/tinymce/js/tinymce/plugins/dotimageclipboard/dialog.jsp [hunk 1]
```diff
@@ -33,13 +33,21 @@ if(clipboard ==null){
 
 <style>
 
+body {
+	margin: 0;
```
Comment (fmontes) [inline dotCMS/src/main/webapp/html/js/tinymce/js/tinymce/plugins/dotimageclipboard/dialog.jsp:37]: We need to make this to use Roboto from the instance of dotcms, see my comment in the PR.

**Review by fmontes — CHANGES_REQUESTED** [review]
.

**Comment by fmontes** [comment]
> > Can you apply the font Roboto so it match the dotCMS font? you can load it from the dotcms instance.

I already set the `Roboto` font in `skin.min.css` file and it's working fine . do you want to load font using specifically path? 

Yes, we can load from the instance, is already there and we can take advantage of the browser cache.

File: dotCMS/src/main/webapp/html/js/tinymce/js/tinymce/plugins/dotimageclipboard/dialog.jsp (hunk 1 above)
Comment (zulqarnainvd) [reply dotCMS/src/main/webapp/html/js/tinymce/js/tinymce/plugins/dotimageclipboard/dialog.jsp:50]: updated

**Pull request merged**